ci/tests/backend/alba_test.py

```diff
@@ -344,42 +348,49 @@ def ovs_3188_verify_namespace_test():
         """
         Verify namespaces
         """
+        required_asds = 1
         nr_of_disks_to_create = 5
         namespace_prefix = 'ovs_3188-'
         compression = 'none'
         encryption = 'none'
         preset_name = 'be_preset_02'
         policies = [[1, 1, 1, 2]]
 
-        backend = GeneralBackend.get_by_name(TestALBA.backend_name)
-        if backend is None:
-            backend = GeneralAlba.add_alba_backend(TestALBA.backend_name).backend
-        GeneralAlba.add_preset(backend.alba_backend, preset_name, policies, compression, encryption)
+        alba_backend = GeneralAlba.get_by_name(TestALBA.backend_name)
+        if alba_backend is None:
+            alba_backend = GeneralAlba.add_alba_backend(TestALBA.backend_name)
+        GeneralAlba.add_preset(alba_backend, preset_name, policies, compression, encryption)
+
+        alba_backend.invalidate_dynamics()
+        claimed_disks = [disk for disk in alba_backend.all_disks if disk['status'] == 'claimed' and disk['alba_backend_guid'] == alba_backend.guid]
+        if len(claimed_disks) < required_asds:
+            uninit_disks = [disk for disk in alba_backend.all_disks if disk['status'] == 'uninitialized']
+            if len(uninit_disks) < required_asds:
+                raise SkipTest('{0} ASD required for this test to be executed, {1} available'.format(required_asds, len(uninit_disks)))
+            disk_names = GeneralAlba.initialise_disks(alba_backend=alba_backend, nr_of_disks=required_asds)
```

Comment:
No because all tests should be independent from each other.
We cannot rely on the fact a previous test already claimed disks or initialized disks
This test needs to have 3 disks initialized and claimed

The check ```if len(claimed_disks) < required_asds:``` should actually be replaced too to something smarter, but due to lack of time i haven't been able to do this